sqs/README.md

```diff
@@ -2,36 +2,237 @@
 
 Current version is `0.1.0`.
 
-
 <a name="server"></a>
 
 ## Server Binding Object
 
 This object MUST NOT contain any properties. Its name is reserved for future use.
 
-
-
 <a name="channel"></a>
 
-## Channel Binding Object
-
-This object MUST NOT contain any properties. Its name is reserved for future use.
-
+## Channel Binding Object 
+
+Use the Channel Binding Operation for Point-to-Point SQS channels.
+
+There are three likely scenarios for use of the Channel Binding Object:
+
+- One file defines both publish and subscribe operations, for example if we were implementing the work queue pattern to offload work from an HTTP API endpoint to a worker process. In this case the channel would be defined on the Channel Object in that single file. 
+- The producer and consumer both have an AsyncAPI specification file, and the producer is raising an event, for example interop between microservices, and the producer 'owns' the channel definition and thus has the SQS Binding on its Channel Object. 
+- The producer and consumer both have an AsyncAPI specification file, and the consumer receives commands, for example interop between microservices, and the consumer 'owns' the channel  definition and thus has the SQS Binding on its Channel Object. 
+
+An SQS queue can set up a Dead Letter Queue as part of a Redelivery Policy. To support this requirement, the Channel Binding Object allows you to define both a Queue Object to use as the Channel or target in a *publish* Operation and a Dead Letter Queue. You can then refer to the Dead letter Queue in the Redrive Policy using the Identifier Object and setting the *name* field to match the *name* field of your Dead Letter Queue Object. (If you define the DLQ externally, the Identifier also supports an ARN). 
+
+### Fields
+|Field Name | Type | Description|
+|---|:---:|---|
+| <a name="channelBindingObjectQueue"></a>`queue` | [Queue](#queue)| **Required.** A definition of the queue that will be used as the channel. |
+| <a name="channelBindingObjectDLQ"></a>`deadLetterQueue` | [Queue](#queue)| **Optional.** A definition of the queue that will be used as the channel. |
+|<a name="channelBindingObjectBindingVersion"></a>`bindingVersion` | string | **Optional**, defaults to `latest`. The version of this binding.|
+
+### Schemas
+
+#### Queue
+|Field Name | Type | Description|
+|---|:---:|---|
+|<a name="queueObjectRef"></a>$ref | `string` | Allows for an external definition of a queue. The referenced structure MUST be in the format of a [Queue](#queue). If there are conflicts between the referenced definition and this Queue's definition, the behavior is *undefined*.|
+| <a name="queueObjectName"></a>`name` | string | **Required.** The name of the queue. When an [SNS Operation Binding Object]() references an SQS queue by name, the identifier should be the one in this field.|
+| <a name="queueObjectType"></a>`type` | string one of: fifo, standard | **Required.**  Is this a FIFO queue or a standard queue? |
```

Comment:
Any reason to use a string field in here instead of a Boolean for `FifoQueue` ? Aws defines it as one: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html